### DISCUSSION TITLE
Welcome to the Swiss SNOMED-Community Discussions!

### DISCUSSION THREAD

**Discussion by SabineK82** [ehealthsuisse/Swiss-SNOMED-Community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by olivier-niederhauser** [comment]
Hi! Just checking who's around. In case you would like to connect: https://www.linkedin.com/in/olivierniederhauser

**Reply by SabineK82** [reply]
Hi how are you?

**Reply by SabineK82** [reply]
Nice to meet you on this page

**Reply by kaicode** [reply]
Hi there Olivier. I sent a connection request.

**Reply by SabineK82** [reply]
oh yeah? can't see it, sorry

**Comment by kaicode** [comment]
@SabineK82 okay for me to create a Show and Tell thread about how to deploy and use the Swiss SNOMED CT Edition using the open source [Snowstorm Lite](https://github.com/IHTSDO/snowstorm-lite) mini FHIR server using the MLDS syndication feed?

**Reply by SabineK82** [reply]
This would be great, it's exactly for what we opened this page, thank you

**Reply by SabineK82** [reply]
Please tell others too

**Reply by kaicode** [reply]
Started here: https://github.com/ehealthsuisse/Swiss-SNOMED-Community/discussions/3
